packages/manager/apps/web/client/app/hosting/general-informations/GENERAL_INFORMATIONS.controller.js

```diff
@@ -239,12 +240,22 @@ export default class HostingGeneralInformationsCtrl {
     );
   }
 
-  retrievingAvailableOffers(productId) {
+  isAvailableOffer(productId) {
     return this.Hosting.getAvailableOffer(productId).then((offers) => {
-      this.availableOffers = offers;
+      if (offers.length > 0) {
+        this.isAvailableOfferOrDetach = true;
+      }
```

Comment:
**nitpick**
```suggestion
      this.isAvailableOfferOrDetach = offers.length > 0;
```

Reply:
isAvailableOfferOrDetach ca also be set to true by isAvailableDetach. I dont want to override it in this case.

Reply:
You're right, this will change the behavior.